controllers/chat/initChatFromProfileAsAnonymousV2.js

```diff
@@ -46,15 +46,6 @@ const initChatFromProfileAsAnonymousV2 = async (req, res) => {
     const channel = client.channel('messaging', {members});
     const createdChannel = await channel.create();
 
-    const {betterChannelMember, betterChannelMemberObject} =
-      await BetterSocialCore.chat.updateBetterChannelMembers(channel, createdChannel, true, {
-        channel_type: CHANNEL_TYPE.ANONYMOUS,
-        anon_user_info_color_code,
-        anon_user_info_color_name,
-        anon_user_info_emoji_code,
-        anon_user_info_emoji_name
-      });
-
     // Default mock chat in case of channel is blocked
     let chat = {
       message: {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code block is missing the `updateBetterChannelMembers` function call that was present in the old hunk. This function seems to be important for updating channel members and should not be removed without a proper replacement.
```diff
+    const {betterChannelMember, betterChannelMemberObject} =
+      await BetterSocialCore.chat.updateBetterChannelMembers(channel, createdChannel, true, {
+        channel_type: CHANNEL_TYPE.ANONYMOUS,
+        anon_user_info_color_code,
+        anon_user_info_color_name,
+        anon_user_info_emoji_code,
+        anon_user_info_emoji_name
+      });
     // Default mock chat in case of channel is blocked
     let chat = {
       message: {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->